src/components/common/Avatar/Avatar.module.css

```
@@ -0,0 +1,7 @@
+.Avatar{
+margin-left: 20px;
+margin-top: 20px;
+width: 150px;
+height: 150px;
```

As we have different usages with different sizes, let's take all available space. We'll define the size on usage.
```suggestion
width: 100%;
height: 100%;
```